.github/workflows/build-occt-with-vcpkg.yml

```
@@ -0,0 +1,170 @@
+          -StorePasswordInClearText `
+          -Name GitHubPackages `
+          -UserName "${{ env.USERNAME }}" `
+          -Password "${{ secrets.GITHUB_TOKEN }}"
+        .$(${{ env.VCPKG_EXE }} fetch nuget) `
+          setapikey "${{ secrets.GITHUB_TOKEN }}" `
+          -Source "${{ env.FEED_URL }}"
+      shell: pwsh
+
+    - name: Add NuGet sources
+      if: runner.os != 'Windows'
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)